local_land_charges_api_stub/documentation/local-land-charges-api.yml

```diff
@@ -54,8 +54,14 @@ paths:
                       type: string
                 charge-type:
                   $ref: '#/definitions/charge-type'
+                charge-sub-category:
+                  $ref: '#/definitions/charge-sub-category'
                 charge-geographic-description:
                   $ref: '#/definitions/charge-geographic-description'
+                charge-addresss:
+                  $ref: '#/definitions/charge-address'
+                registration-date:
+                  $ref: '#/definitions/registration-date'
```

Comment:
We don't allow registration date to be specified on 'add charge', it gets auto filled.